File: 백준/Bronze/5086. 배수와 약수/배수와 약수.py


while True:
    n, m = map(int, input().split())
    if n == 0 and m == 0:
        break
    else:
        if n%m ==0 and n>m:
            print('multiple')
        elif m%n ==0 and n<m:
            print('factor')
        else:
            print('neither')

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        while (scan.hasNext()){
            int a = scan.nextInt();
            int b = scan.nextInt();
            if(a == 0 && b == 0) {
                break;
            }
            if(a%b == 0){
                System.out.println("multiple");
            } else if (b%a == 0) {
                System.out.println("factor");
            } else {
                System.out.println("neither");
            }

        }
    }
}
